### PR TITLE
5.2021.8 Release Notes

### PR DESCRIPTION
**Pull request by kalinchan**

### CODE REVIEW

File: docs/modules/ROOT/pages/release-notes/release-notes-2021-8.adoc [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+#### Breaking Changes
+
+##### Client Certificate Validation Checks
+
+After upgrading to Payara Community 5.2021.8,  a certificate that was accepted in earlier versions might be rejected now. It is no longer enough that the client certificates are included within the Payara Keystore. See [Certificate Expiration Validation :: Payara Community Documentation](https://docs.payara.fish/community/docs/documentation/payara-server/server-configuration/security/certificate-realm-certificate-validation.html#_client-certificate-expiration-validator)
+
+### Improvements
+
+[[FISH-5687](https://github.com/payara/Payara/pull/5394)] Integrate HotSwap Agent in Payara Platform
```
Comment (cubastanley) [inline docs/modules/ROOT/pages/release-notes/release-notes-2021-8.adoc:19]: Links need to be moved in front of issue tags
```suggestion
[(https://github.com/payara/Payara/pull/5394)[FISH-5687]] Integrate HotSwap Agent in Payara Platform
```

Comment (cubastanley) [reply docs/modules/ROOT/pages/release-notes/release-notes-2021-8.adoc:19]: cc @debbiehoffman @rdebusscher - I'll update the internal docs for how this should be done

File: docs/modules/ROOT/pages/release-notes/release-notes-2021-8.adoc [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+[https://github.com/payara/Payara/pull/5411[FISH-5736]] Fix Unknown Exclude Field Provided Warning on Startup
+
+[https://github.com/payara/Payara/pull/5431[FISH-5734]] Recursive Update Exception when Reading a MicroProfile Config Value After Server Restart
+
+[https://github.com/payara/Payara/pull/5420[FISH-5724]] Deployment Failure Due to  'The lifecycle method [postConstruct] Must Not Throw a Checked Exception'
+
+[https://github.com/payara/Payara/pull/5408[FISH-5678]] Jakarta Named Properties in 'persistence.xml' Are Not Recognized
+
+[https://github.com/payara/Payara/pull/5440[FISH-5675]] (Community Contribution - https://github.com/PhillipRoss95/PhillipRoss[phillipross]) Full State Saving with Mojarra Results in ArrayIndexOutOfBoundsException
```
Comment (OndroMih) [inline docs/modules/ROOT/pages/release-notes/release-notes-2021-8.adoc:51]:
```suggestion
[https://github.com/payara/Payara/pull/5440[FISH-5675]] (Community Contribution - https://github.com/PhillipRoss95[phillipross]) Full State Saving with Mojarra Results in ArrayIndexOutOfBoundsException
```

**Comment by rdebusscher** [comment]
Already merging this as I need to adapt the release notes a bit.

**Pull request merged**